lua/crates/popup/dependencies.lua

```diff
@@ -153,7 +153,7 @@ function M.open_deps(ctx, crate_name, version, opts)
    local title = string.format(state.cfg.popup.text.title, crate_name .. " " .. version.num)
    local deps_width = 0
    local deps_text_index = {}
-
+   local HlTextDepList = {}
```

Comment:
How did that end up here? What version of the `tl` compiler are you using?

Reply:
This was the first time I heard of teal at all. I just installed things until all `make` targets worked without errors. `tl --version` reports `0.13.2` and I also installed the `inspect`, `compat53`, `argparse` and `luafilesystem` packages with `sudo luarocks install <package name>`.

Reply:
Ok, mine says `0.14.1`, just reinstalling `tl` via `luarocks` should do the trick.

Reply:
Apparently the [AUR package](https://aur.archlinux.org/packages/teal) of teal is outdated and I didn't notice. I now installed `tl` through luarocks and added a commit with the new compiler version.